sqlitecluster/SQLite.cpp

```diff
@@ -53,22 +61,36 @@ SQLite::SQLite(const string& filename, int cacheSize, int autoCheckpoint, bool r
     // does not need to wait on locks created in this thread.
     SASSERT(sqlite3_enable_shared_cache(0) == SQLITE_OK);
 
+    // Set our journal table name.
+    _journalName = getJournalTableName(journalTable);
+
     // Open or create the database
     if (SFileExists(_filename)) {
         DBINFO("Opening database '" << _filename << "'");
     } else {
         DBINFO("Creating new database '" << _filename << "'");
     }
-    if (!readOnly) {
-        // Open a read/write database
-        SASSERT(!sqlite3_open_v2(filename.c_str(), &_db, DB_WRITE_OPEN_FLAGS, NULL));
-        _setPragmas(_db, autoCheckpoint);
-        if (SQVerifyTable(_db, "journal", "CREATE TABLE journal ( id INTEGER PRIMARY KEY, query TEXT, hash TEXT )")) {
-            SHMMM("Created journal table.");
+    // Open a read/write database
```

Comment:
Please insert a space between previous blocks of code and the next.  The pattern is:

```
// This is a comment
thisIsCode();

// Note the space above
moreCode();
```